stable/spark-operator/templates/serviceaccount.yaml

```diff
@@ -4,8 +4,6 @@ kind: ServiceAccount
 metadata:
   name: {{ include "spark-operator.serviceAccountName" . }}
   annotations:
-    "helm.sh/hook": pre-install
-    "helm.sh/hook-delete-policy": hook-failed, before-hook-creation
```

Comment:
👏🏼

Reply:
Could you please use words? I don't speak middle-schoolish